src/ragger/conftest/base_conftest.py

```diff
@@ -153,9 +153,12 @@ def prepare_speculos_args(root_pytest_dir: Path, firmware: Firmware, display: bo
 def create_backend(root_pytest_dir: Path, backend_name: str, firmware: Firmware, display: bool,
                    log_apdu_file: Optional[Path], cli_user_seed: str):
     if backend_name.lower() == "ledgercomm":
-        return LedgerCommBackend(firmware=firmware, interface="hid", log_apdu_file=log_apdu_file)
+        return LedgerCommBackend(firmware=firmware,
+                                 interface="hid",
+                                 log_apdu_file=log_apdu_file,
+                                 with_gui=True)
```

Comment:
```suggestion
                                 with_gui=display)
```
What do you think of reusing this parameter?

Reply:
Makes sense. I'll also add some doc to explain this, it could be confusing in a way.